streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1775,30 +1286,9 @@ public void process(final Record<String, String> record) {
         }
     }
 
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
-    private <K, V> org.apache.kafka.streams.processor.ProcessorSupplier<K, V> define(final org.apache.kafka.streams.processor.Processor<K, V> processor) {
-        return () -> processor;
-    }
-
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
-    private <K, V> org.apache.kafka.streams.processor.ProcessorSupplier<K, V> defineWithStoresOldAPI(final Supplier<org.apache.kafka.streams.processor.Processor<K, V>> supplier,
```

Comment:
There is already `defineWithStores()` for new PAPI -- nothing to be migrated -- deleting